doc/getting-started.md

```diff
@@ -19,7 +21,24 @@ This guide will help you get started writing contracts with CTL. Please also see
 
 ## Prerequisites
 
-You need to have set up a Purescript project using CTL as a dependency ([more details here](./ctl-as-dependency.md)). You will also need to become familiar with [CTL's runtime](./runtime.md) as its runtime services are required for executing virtually all contracts.
+### Setting up a new project
+
+The easiest way to create a new CTL project is to use our `ctl-scaffold` flake template. This lives in the CTL repo -- you can have a look [here](../templates/ctl-scaffold). It contains a simple, yet complete, flakes-based scaffolding project with example `outputs` for a CTL project, including its runtime.
+
+A new project can be initialized as follows:
+
+```
+$ mkdir new-project && cd new-project
+$ nix flake init -t github:Plutonomicon/cardano-transaction-lib
+$ git init
```

Comment:
I think we should not bother ourselves with documenting how to use git

Reply:
I think it's important to include git in the directions for two reasons:
- `nix flake init` just dumps all of the template files in the working directory but doesn't create a repo, which is kind of problematic because
- flakes are just source repositories that contain a `flake.nix`. That is, they need to be repos. In the past, several people have wondered why local files are invisible to the flake because they hadn't added them with git yet

So I feel that omitting the git part leaves an incomplete example that users who aren't as familiar with flakes will be possibly misled by

Reply:
Agree.

Reply:
Then let's add a note that `git` is here for a reason.